src/scripts/view.js

```diff
@@ -309,9 +309,12 @@ view.photo = {
 		$('body').css('overflow', 'hidden')
 
 		// Fullscreen
-		$(document)
-			.bind('mouseenter', header.show)
-			.bind('mouseleave', header.hide)
+		var timeout
+		$(document).bind('mousemove', function() {
+			clearTimeout(timeout)
+			header.show()
```

Comment:
Yes, that's really unnecessary. There was another call of header.hide for mobile, but it had delay=0, so that we didn't need the timeout there either. I've removed that now.